frontend/lib/configuration/first_start.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:shared_preferences/shared_preferences.dart';
+
+Future<bool> isFirstStart() async {
+  try {
+    final sharedPreferences = await SharedPreferences.getInstance();
+    if (sharedPreferences.containsKey("firstStart")) {
+      return false;
```

Comment:
We could think about returning the actually stored value here, just to reduce confusion in case someone for example sets the value to true in his shared prefs for debugging purposes or something.